scripts/curio-apt/DEBIAN/control

```diff
@@ -0,0 +1,20 @@
+Source: curio
+Section: Network
+Priority: optional
+Maintainer: Andrew Jackson <support@curiostorage.org>
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.6.0
+Homepage: github.com/filecoin-project/lotus
+Rules-Requires-Root: no
+Package: $PACKAGE
+Version: $VERSION
+Architecture: amd64
+Depends: hwloc, mesa-opencl-icd
+Description: A Filecoin Storage Provider implementation.
+ This Filecoin Storage Provider implementation is a fork of the Lotus project.
+ It improves upon the previous by solving more complete problem sets including
+ avoiding single points of failure, improving performance, and providing a
+ more robust and scalable solution.
+ It needs to be run in conjunction with the Filecoin network daemon and requires
+ setup of a Miner-ID and a Wallet-ID. See documentation on the website for more
```

Comment:
Maybe provide link to curiostorage.org or [lotus.filecoin.io](https://lotus.filecoin.io/storage-providers/curio/overview/) ?